docs/bridge/gb-testnet.md

```diff
@@ -0,0 +1,161 @@
+|  WBTC | 0xC04B0d3107736C32e19F1c62b2aF67BE61d63a05 | 0x7825cB7feEAD896241f748c89550F3D01AF51e48 |
+|  DAI  | 0xdc31Ee1784292379Fbb2964b3B9C4124D8F89C60 | 0x71339a9C403383c3E18712130615d369Ff9a7124 |
+
+
+#### Currently supported wallets:
+- Metamask
+
+We are constantly working on adding new tokens and chain support. If you have any feedback and issues, please drop us an email at product.bridge@crypto.org.
+
+### How to use the Gravity Bridge [Web Dapp]
+
+#### 1. Connect your wallet
+Click the “Connect Wallet’ button to connect your desktop or browser extension wallet. We currently support browser-compatible versions of the Metamask. Click connect wallet and look for a popup from your wallet interface or click into the wallet extension to give consent.
+
+<img src="./assets/webapp_connect_wallet.png"  alt="centered image" />
+
+ **Testnet**
+If you are using Metamask, make sure to enable test networks, on the Metamask wallet 
+Go to settings > Advanced > enable `Show test networks`. Now you should be able to select the Goerli Test Network in the list of networks, which we will be using in this tutorial.
+
+<img src="./assets/testnetworks.png" width="300"  alt="centered image"/>
+
+
+For the Pioneer testnet, Go to settings > Networks > Add network 
+Add the following network details:
+
+
+- Network Name: Pioneer11
+- New RPC URL: https://evm-p11.cronos.org
+- ChainID: 340
+- Currency symbol: TCRO
+- Block Explorer URL (Optional): https://cronos.crypto.org/explorer/pioneer11
+
+
+<img src="./assets/metamaskpioneer11.png" width="300"  alt="centered image"/>
+
+
+#### 2. Select Network and Token
+Select the origin chain on the left and the destination chain on the right in the bridge interface. Auto-suggestion may set some parameters. However, a manual check might be needed to match your wallet settings to the selected network. Once the networks are chosen, select the asset you would like to transfer. 
+Note that your ERC20 compatible addresses on Ethereum and Cronos are the same.
+
+
+<img src="./assets/pioneerassets.png"   alt="centered image"/>
+
+
+If you would like to swap any of the other tokens listed, you can head to [Uniswap](https://app.uniswap.org/#/swap?chain=goerli) on Goerli testnet. Refer to the appendix for an example.
+
+#### 3. Enter the amount
+After selecting the network and token, enter and confirm the amount you would like to transfer.
+The bridge network fees will be calculated accordingly. You always need to pay the gas fee charged by the source network. Before bridging a large amount, we encourage testing a transfer of a minor amount first to ensure all the settings are correct.
+
+If this is the first time you are swapping this token using the Gravity Bridge, you may need to enable the contract such that the Gravity Bridge has access to this token. Accept Permission in your wallet.
+
+
+<img src="./assets/enablecontract.png"   alt="centered image"/>
```

Comment:
(Optional) The below examples with images are about eth/weth/TCRO transfer, in this image it was USDT, it may not looks super consistent.

Reply:
updated, thing is the enabling of usdc is needed, whereas eth it usually will not be needed